runserver.py

```diff
@@ -7,4 +7,4 @@
 app.config['APPLICATION_ROOT'] = '/'
 
 if __name__ == '__main__':
-    app.run(debug=True, host=0.0.0.0,  port=8100)
+    app.run(debug=True, host='0.0.0.0',  port=8100)
```

Comment:
Isn't a default 0.0.0.0 development IP a security risk?

Reply:
Why is it necessary with the host IP there?

Reply:
The default is 127.0.0.1 (localhost). The need to add host 0.0.0.0 is to bind to all interfaces.
This is relevant for Docker, otherwise, it doesn't work.
We can also externalize this to the config file. It would make sense.
What do you think @fnielsen ?

Reply:
Default on localhost and then perhaps configurable.

Reply:
Done as suggested.